docs/001_develop/03_client-capabilities/023_custom-components/05_directives.mdx

```diff
@@ -184,24 +179,28 @@ export class MyButton extends GenesisElement {
 </my-button>
 ```
 
->- `childNodes` will contain everything inside `<div>`, including `<span>` elements, text nodes like "Text node", and `<div>` elements.
+- `childNodes` will contain everything inside `<div>`, including `<span>` elements, text nodes like "Text node", and `<div>` elements.
 
->- Any changes in the slot content (e.g., adding or removing nodes) will trigger `childNodesChanged` and log the updated list.
+- Any changes in the slot content (e.g., adding or removing nodes) will trigger `childNodesChanged` and log the updated list.
 
 #### Syntax breakdown
 
 ```typescript
 children(propertyName, options);
 ```
 
->- `propertyName`: Specifies the property that will store the array of child nodes.
->- `options` (optional): Additional settings for filtering or structuring child nodes. This can include:
-  >>- `filter`: Restricts the collection to specific elements (e.g., elements('span')).
-  >>- `positioning`: Includes index and other contextual properties if true.
+- `propertyName`specifies the property that will store the array of child nodes.
+- `options` (optional)provides additional settings for filtering or structuring child nodes. This can include:
+  - `filter`restricts the collection to specific elements (e.g., elements('span')).
+  - `positioning` includes index and other contextual properties if true.
```

Comment:
```suggestion
- `propertyName`specifies the property that will store the array of child nodes.
- `options` (optional) provides additional settings for filtering or structuring child nodes. This can include:
  - `filter` restricts the collection to specific elements (e.g., elements('span')).
  - `positioning` includes index and other contextual properties if true.
```